src/v2/contexts/overlayContext/OverlayContext.tsx

```diff
@@ -24,11 +45,15 @@ import type {
   DefaultUserType,
   UnknownType,
 } from '../../types/types';
+import type { Streami18n } from '../../utils/Streami18n';
 
 export type Overlay = 'none' | 'gallery' | 'message';
 
 export type OverlayContextValue = {
   overlay: Overlay;
+  setBlurType: React.Dispatch<
+    React.SetStateAction<'light' | 'dark' | undefined>
```

Comment:
nbd but maybe create a local type for BlurType and use it here and within the useState